landoui/app.py

```diff
@@ -42,7 +43,19 @@ def create_app(
     session_cookie_name, session_cookie_domain, session_cookie_secure,
     use_https
 ):
+
+    csp = {
```

Comment:
I know the formatter did this, do you think there is anyway to fix it? Don't worry about it if it looks like a timesink.